app/controllers/submit_proposals_controller.rb

```diff
@@ -11,6 +11,7 @@ def create
     update_proposal_ams_subject_code
     submission = SubmitProposalService.new(@proposal, params)
     submission.save_answers
+    @proposal.allow_late_submission = true unless @proposal.draft?
```

Comment:
This would allow late submissions from all proposals that are not in draft state. What we want is to allow late submissions _only for proposals that have requested revision_.